chain/Makefile

```diff
@@ -1,8 +1,13 @@
 PACKAGES=$(shell go list ./... | grep -v '/simulation')
 VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
 COMMIT := $(shell git log -1 --format='%H')
+LEDGER_ENABLED ?= true
 BINDIR ?= $(GOPATH)/bin
 
+ifeq ($(LEDGER_ENABLED),true)
+	build_tags += ledger
```

Comment:
Use UPPER_CASE like others?

Reply:
I follow the old convention to minimize change. See code below:

`BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'`